src/server/ast/symbolTable.ts

```diff
@@ -2,29 +2,38 @@ import {Parser} from '../parser/parser'
 
 export enum SymbolTypes
 {
-	invalid = 0x0000,
-	integer = 0x0001,
-	signed = integer | 0x0000,
-	unsigned = 0x0002,
-	int8Bit = integer | 0x0000,
-	int16Bit = integer | 0x0004,
-	int32Bit = integer | 0x0008,
-	int64Bit = integer | 0x000C,
-	character = 0x0010,
-	list = 0x0020,
+	invalid = 0x000000,
+	integer = 0x000001,
+	signed = integer | 0x000000,
+	unsigned = 0x000002,
+	int8Bit = integer | 0x000000,
+	uint8Bit = integer | 0x000002,
+	int16Bit = integer | 0x000004,
+	uint16Bit = integer | 0x000006,
+	int32Bit = integer | 0x000008,
+	uint32Bit = integer | 0x00000A,
+	int64Bit = integer | 0x00000C,
+	uint64Bit = integer | 0x00000E,
+	float = 0x000010,
+	float32Bit = float | 0x000000,
+	float64Bit = float | 0x000004,
+	character = 0x000020,
+	list = 0x000040,
 	string = character | list,
-	struct = 0x0040,
+	struct = 0x000080,
 	dict = struct | list,
-	array = 0x0080,
+	array = 0x000100,
 	set = struct | array,
-	bool = 0x0100,
-	function = 0x0200,
-	reference = 0x0400,
-	pointer = 0x0800,
-	pack = 0x1000, // This is for template type/value packs
-	auto = 0x2000,
-	none = 0x4000,
-	type = 0x8000
+	bool = 0x000200,
+	enum = 0x000400,
+	class = 0x00800,
```

Comment:
`class` is `struct` above as while the keyword is `class`, the language defines them as structures-with-members so internally we've been using `struct`